lib/public/app.css

```diff
@@ -162,6 +162,10 @@ th.text-center, td.text-center {
     text-align: center;
 }
 
+.table-cl-mh-2 {
+    max-height: 48px;
+}
+
```

Comment:
You should avoid as much as possible to use fixed pixels height. I would rather suggest to use `line-clamp` which seems to be widely available (with `-webkit-` prefix) https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp#truncating_a_paragraph

Reply:
I tried using this, but due to the fact that all statuses are separate components instead of one text it is not possible to use this

Reply:
Ok, that makes sense. In that case, please use `em` or `rem` at least.
Also, use more explicit class: `table-cl-mh-2` is completely opaque to me